### PR TITLE
Style it

### PR DESCRIPTION
**Pull request by jcorteza**

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    background-color: #e9e9e9;
+}
+
+#apiDataContainer a, #apiDataContainer a:visited {
+    color: black;
+    text-decoration: none;
+}
+
+.tweetContainer img {
+    float: left;
```
Comment (kev-chan) [inline src/css/main.css]: if you use floats, you should add specific css afterwards to stop the floating

File: src/css/main.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+.handle {
+    font-size: 8pt;
+}
+
+.tweetContainer img {
+    max-width: 75px;
+    height: auto;
+    margin: 10px 0px;
```
Comment (kev-chan) [inline src/css/main.css]: you can just say 0 vs 0px

File: src/js/functional.js [hunk 3]
```diff
@@ -31,27 +31,36 @@ document.addEventListener("DOMContentLoaded", () => {
             const newImage = document.createElement("img");
             const newPargraph = document.createElement("p");
```
Comment (kev-chan) [inline src/js/functional.js:31]: you could use a div instead of a p tag in all areas

File: src/js/functional.js [hunk 4]
```diff
@@ -31,27 +31,36 @@ document.addEventListener("DOMContentLoaded", () => {
...
-            if(statusArray[i].user) {
+            if(user) {
                 newImage.src = statusArray[i].user.profileImageUrl;
                 anchor.href = statusArray[i].postUrl;
+                nameText.textContent = user.name;
+                handleText.textContent = user.twHandle;
+                userDiv.append(handleText);
             } else {
                 newImage.src = "./img/twitter-logo.png";
                 anchor.href = "";
```
Comment (kev-chan) [inline src/js/functional.js:50]: why is the href separated in the user conditional

File: src/css/main.css [hunk 5]
```diff
@@ -0,0 +1,77 @@
+* {
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    font-family: 'Trebuchet MS', Arial, sans-serif;
+    padding: 2% 5%;
```
Comment (kev-chan) [inline src/css/main.css]: i'd recommend using specific values over percentages

**Review by kev-chan — CHANGES_REQUESTED** [review]

File: src/js/functional.js (hunk 3 above)
Comment (jcorteza) [reply src/js/functional.js:31]: using tags like p are better for accessibility, things like screen readers

File: src/js/functional.js (hunk 4 above)
Comment (jcorteza) [reply src/js/functional.js:50]: The url includes the user's screen name. If there's no user there would be no screen name, no proper url

**Pull request merged**